test/integration/e2e.spec.ts

```diff
@@ -114,6 +114,30 @@ describe('e2e', function () {
       expect((await feiDAO.votingDelay()).toString()).to.be.equal('10');
       expect((await contracts.daiBondingCurve.duration()).toString()).to.be.equal('11');
     });
+
+    it('rollback succeeds', async function() {
+      const { feiDAO, timelock, governorAlpha } = contracts;
+      const { multisig } = contractAddresses;
+
+      const signer = await ethers.getSigner(multisig);
```

Comment:
I *believe* this is out of order; I think you need to impersonate the account and then get the signer. Could be wrong.

Either way if you rebase off of master you can use the new helper method I just added.

Reply:
will use going forward